src/components/Layout/Header/Header.tsx

```diff
@@ -261,6 +258,9 @@ const HeaderContent: FunctionComponent<Props & { open: boolean; sticky: boolean;
                     <div className="ml-3 !mt-2 flex flex-col items-start space-y-4">{callToAction}</div>
                 </div>
             </Disclosure.Panel>
+            <Modal open={isSignUpModalOpen} handleClose={closeModal}>
```

Comment:
To confirm, this is the only meaningful change, right? The rest of this PR is just linting?

Reply:
If so, feel free to merge, this makes sense